010.md

```diff
@@ -0,0 +1,451 @@
+# Thread Spec
+
+Authors: Mix Irving mix@protozoa.nz
+
+Date: 2023-03-27
+
+License: CC0-1.0
+
+The `post` message type is the foundational unit for talking with peers on
+the original scuttlebutt network. 
+
+`post` messages can be used standalone, but are most commonly collected as
+sets of related messages and formed into a "thread".
+
+During rendering, threads of posts are also further _decorated_ with `vote`
+messages (for likes), and `about` messages (for displaying author details).
+
+
+## 1. `post` schema
+
```

Comment:
```suggestion

The following schemas relate to the `content` field in SSB messages. If a field is commented with `// optional`, it means it MAY appear in the `content` field, otherwise it SHOULD appear.

```